### PR TITLE
CO/2 description from Benzene example

### PR DESCRIPTION
**Pull request by tcolehunter**

### CODE REVIEW

**Review by kbilsback — COMMENTED** [review]
I've been working on a cleaned up version of this for qaqc based on Kristen's plots, so I think this merge will be redundant.

File: Hello.R [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello
```
Comment (kbilsback) [inline Hello.R:1]: This shouldn't be merged to the master

File: r_markdown/benzene_emissionsfactor_qa.Rmd [hunk 2]
```diff
@@ -0,0 +1,170 @@
+---
+title: "VOCs Emissions Factor QA/QC"
```
Comment (kbilsback) [inline r_markdown/benzene_emissionsfactor_qa.Rmd:2]: I'm working on an cleaned up emissions factor qaqc that would make this file redundant

File: r_markdown/test_co.Rmd [hunk 3]
```diff
@@ -0,0 +1,164 @@
+---
+title: "Test CO/2"
```
Comment (kbilsback) [inline r_markdown/test_co.Rmd:2]: same thing

File: r_markdown/toluene_emissionsfactor_qa.Rmd [hunk 4]
```diff
@@ -0,0 +1,173 @@
+---
+title: "Toluene Emissions Factor QA/QC"
```
Comment (kbilsback) [inline r_markdown/toluene_emissionsfactor_qa.Rmd:2]: same thing

File: Hello.R (hunk 1 above)
Comment (tcolehunter) [reply Hello.R:1]: Fair enough! All hail Master Good :+1:

**Pull request closed without merging**